specification/reservations/resource-manager/readme.md

```diff
@@ -29,6 +29,17 @@ openapi-type: arm
 tag: package-2020-10-preview
```

Comment:
Would you want to change the default tag to your new tag?

Reply:
I've updated the default tag now.